### ISSUE TITLE
‘ls -l’ as well as less/more does not work fine on iOS 15.0

### ISSUE THREAD

**Issue by abelli5** [ish-app/ish#1389]
I upgrade my iPad 2020 to 15.0 as well as iSH to latest version. But now it does not work fine when I enter the commands such as ‘ls -l’, it hangs or only give summary without files list any more.
The other commands such as ‘less’ / ‘more’ perform same as ‘ls -l’. It seems some extra characters result in the issue.

**Comment by 62f** [comment]
Can you please provide a screen shot showing that iOS 15.x.x is installed? Thanks!

**Comment by ylytkin** [comment]
I too noticed strange behavior with ls -l, in my case quite (not always, but quite frequently) ls -l does not give a full list of items in the directory. I ran a test, here the Test dir contains directories 1 to 5, attaching the result below. I'm on iOS 15.2 (19C56).

![0CC9C1AD-49A7-48D7-96E5-6F662A93EFD7](https://user-images.githubusercontent.com/43873680/148049855-9ce732a3-9716-4e60-a3d8-af4264a045ec.png)

**Comment by Rhinozz** [comment]
ylytkin, I'm unable to reproduce this in both ash and bash. What shell are you using and how did you install it?

**Comment by saagarjha** [comment]
Perhaps you are on a TestFlight build? We put https://github.com/ish-app/ish/commit/4b78de2befd30fb7bd6e4c019e5b2b2626106bbd in recently to address an issue with symptoms similar to this.

**Comment by ylytkin** [comment]
Yes, I actually was on an AppStore version of iSH when I wrote this, but after that I saw comments about this in a different issue here and installed a TestFlight version after that. Can confirm the issue with ls seems to be gone now!

**Comment by Rhinozz** [comment]
> Perhaps you are on a TestFlight build? We put [4b78de2](https://github.com/ish-app/ish/commit/4b78de2befd30fb7bd6e4c019e5b2b2626106bbd) in recently to address an issue with symptoms similar to this.

Correct! The fix seems to work for OP.

**Comment by Rhinozz** [comment]
> Perhaps you are on a TestFlight build? We put [4b78de2](https://github.com/ish-app/ish/commit/4b78de2befd30fb7bd6e4c019e5b2b2626106bbd) in recently to address an issue with symptoms similar to this.

Correct! The fix seems to work for OP.

**Comment by sno0pastaaa** [comment]
cd ..
